io.github.dummerle.rare.json

```
@@ -0,0 +1,48 @@
+{
+    "app-id": "io.github.dummerle.rare",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/rare",
+    "finish-args": [
```

```suggestion
    "rename-icon": "rare",
    "finish-args": [
```

This assuming they are installed by the python wheel.